RELEASE.md

```diff
@@ -1,5 +1,6 @@
 # (unreleased)
 - Improved support for secondary trigonometric functions such as sec and acoth.
+- Improved printing of devisions For example `1 / (1/cos(x))` now gets rendered as `cos(x)` whereas previously it would be `1 / 1 / cos(x)` giving the incorrect result. An side effcet of the change is that powers of formulas get extra brackets e.g. `pow((1 / x), 2)`
```

Comment:
Should probably mention this is a cellmlmanip change, not a chaste codegen change? And maybe raise the required version number then? Otherwise I can use this change codegen with an older cellmlmanip and _not_ see the change